sortPeople.go

```diff
@@ -0,0 +1,54 @@
+package main
+
+import (
+	"fmt"
+	"os"
+	"sort"
+	"time"
+)
+
+type Person struct {
+	firstName string
+	lastName  string
+	birthDay  time.Time
+}
+
+func (p Person) String() string {
+	return fmt.Sprintf("{%s %s %s}", p.firstName, p.lastName, p.birthDay.Format("2006-01-02"))
+}
+
+type People []Person
+
+func (p People) Len() int {
+	if len(p) <= 0 {
+		fmt.Println("There are no people")
+		os.Exit(1)
+	}
+	return len(p)
+}
+func (p People) Less(i, j int) bool {
+	if p[i].firstName == p[j].firstName {
+		return p[i].String() > p[j].String()
```

Comment:
String used more for console pretty print. I'd like to avoid use this function there